src/radical/pilot/agent/agent_0.py

```diff
@@ -325,25 +325,21 @@ def _start_sub_agents(self):
                 #        work with a number of launch methods.  Can the
                 #        offset computation be moved to the LRMS?
                 ls_name = "%s/%s.sh" % (os.getcwd(), sa)
-                slots = {
```

Comment:
The sub-agent startup went out of sync with the scheduler changes.  This fix is needed for running jsrun from the compute nodes.